molgenis-core-ui/src/main/java/org/molgenis/core/ui/MolgenisWebAppInitializer.java

```diff
@@ -58,6 +59,12 @@ protected void onStartup(ServletContext servletContext, Class<?> appConfig, int
     }
 
     // add filters
+    Dynamic forwardedHeaderFilter =
+        servletContext.addFilter("forwardedHeaderFilter", ForwardedHeaderFilter.class);
+    forwardedHeaderFilter.setAsyncSupported(true);
+    forwardedHeaderFilter.addMappingForUrlPatterns(
+        EnumSet.of(DispatcherType.REQUEST, DispatcherType.ASYNC), false, "*");
```

Comment:
What happens if I create an endpoint ```/test``` that returns ```forward:/test-forwarded``` that returns the current request URL. This determines whether you need to add DispatcherType.FORWARD to this list (and maybe others have well?).

Reply:
Tested like this:
```java
  @GetMapping(value = "/test")
  public ModelAndView forward() {
    return new ModelAndView(format("forward:%s/test-forwarded", BASE_URI));
  }

  @GetMapping(value = "/test-forwarded")
  @ResponseBody
  public String forwarded(HttpServletRequest request) {
    return ServletUriComponentsBuilder.fromCurrentRequest().toUriString();
  }
```

Doing a request with `X-Forwarded-Proto:https` gives: `"http://localhost:8080/api/v1/test-forwarded"`